pkg/META.in

```diff
@@ -53,7 +53,7 @@ package "eq" (
   version = "%{version}%"
   description = "[@@deriving eq]"
   requires(-ppx_driver) = "ppx_deriving"
-  ppxopt(-ppx_driver,-custom_ppx) = "ppx_deriving,./ppx_deriving_eq.cma"
+  ppxopt(-ppx_driver,-custom_ppx) = "ppx_deriving,-deriving-plugin ./ppx_deriving_eq.cma"
```

Comment:
Does this actually work? I don't recall how the findlib expansion works exactly but it may be looking for a path at the start of the line.